examples/app-with-networking/main.tf

```diff
@@ -1,6 +1,6 @@
 resource "opentelekomcloud_compute_keypair_v2" "terraform" {
   name       = "terraform"
-  public_key = "${file("${var.ssh_key_file}.pub")}"
+  public_key = file("${var.ssh_key_file}.pub")
```

Comment:
`"${var.ssh_key_file}.pub"` miss here

Reply:
No, this is valid place for using interpolation